### DISCUSSION TITLE
Add folders/groups for better management of a large number of accounts

### DISCUSSION THREAD

**Discussion by ekos2001** [Authenticator-Extension/Authenticator#997] [Ideas]
It would be good to have folders or groups for better management of a large number of accounts

**Comment by PabloFaccianoGlobant** [comment]
Hi
Please we need folders to organize accounts.
Lets say I have 50 accounts, so scrolling or search is not easy. Maybe folders can help, so I can split account by different project or personal account/busines accounts.
Thanks

**Comment by GuillaumeVrill** [comment]
Hi !

I totally agree, more likely as an IT expert because we manage a lot of customer accounts, team tools and so on.
We also have more and more customers demands on this stuff because of the general security upgrade we see everywhere on SAAS.

In my opinion, having groups or folders to rearrange all those keys would really add value to it at the moment, as correctly naming them is not enough anymore.
